pom.xml

```diff
@@ -51,8 +52,8 @@
     <properties>
         <!-- Instruct the build to use only UTF-8 encoding for source code -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
I don't think we should require Java 11 at *runtime* just yet, as it quite drastically reduce the number of potential users (many are still on Java 8 for whatever reasons). At this point, I'd prefer to make sure that Debezium can be *built* with JDK 11 (and here I'm fine with requiring JDK 11 as a baseline, as the number of contributors is much smaller than the number of users).

Reply:
Yeah that’s a good point! I’ll update that!

- [ ] Target java 8 for runtime still